python/lsst/sconsUtils/scripts.py

```diff
@@ -102,15 +109,26 @@ def initialize(cls, packageName, versionString=None, eupsProduct=None, eupsProdu
             except TypeError:
                 pass
             state.targets["version"] = state.env.VersionModule(versionModuleName)
+        scripts = []
         for root, dirs, files in os.walk("."):
             if "SConstruct" in files and root != ".":
                 dirs[:] = []
                 continue
             dirs[:] = [d for d in dirs if not d.startswith('.')]
             dirs.sort()  # happy coincidence that include < libs < python < tests
```

Comment:
Good catch; I'd meant to remove that whole line (and I will now).  It should not be relevant.